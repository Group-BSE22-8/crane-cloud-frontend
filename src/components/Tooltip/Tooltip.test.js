import React from "react";

import { shallow } from "enzyme";

import Tooltip from "./index";

describe('Test the component', () => {
    it('checks if tooltip matches snapshot', () => {
        const TooltipIcon = shallow(<Tooltip showIcon={true} keyword="" position="top" message="" />);
        const TooltipNoIcon = shallow(<Tooltip showIcon={false} keyword="" position="top" message="" />);
        expect(TooltipIcon).toMatchSnapshot();
    })
})