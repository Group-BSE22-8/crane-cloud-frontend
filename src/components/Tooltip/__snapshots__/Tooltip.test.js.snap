// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the component checks if tooltip matches snapshot 1`] = `ShallowWrapper {}`;
